apps/molgenis-components/src/components/forms/RowEdit.vue

```diff
@@ -102,7 +102,7 @@ export default {
     },
     errorPerColumn: {
       type: Object,
-      default: () => {},
+      default: () => ({}),
```

Comment:
wow this is tricky business. But indeed otherwise it is a function instead of an object.